test/ret_web/controllers/api-internal/change_login_email_controller_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule RetWeb.ApiInternal.V1.ChangeLoginEmailControllerTest do
+  use RetWeb.ConnCase
+  import Ret.TestHelpers
```

Comment:
nit: Consider separating `use` from `alias`, `import`, and `require`. The former injects code, the latter are just types of imports.